packages/site-kit/components/Nav.svelte

```diff
@@ -20,10 +20,15 @@
 		open = false;
 	});
 
+	let window_inner_width;
+	function is_mobile() {
```

Comment:
I think it'd be better to call this `is_small_screen` or something. Because it could be triggered on desktop as well

Reply:
Sounds good. My only other thought was to shift the menu down so the links don't overlap (like the mockup below) - that would remove the need for all this code and the `preventDefault`, and allow us to use the passive event modifier on the scroll listener that had previously caused an issue in #33 (which was added to address an issue in the Lighthouse audit).

If you'd prefer I can do that instead - otherwise I'll just make the change to `is_small_screen` and wrap it up.

![kit-mockup](https://user-images.githubusercontent.com/10662340/117285032-dd92f080-aeaa-11eb-8388-e1504d75f378.png)